rskj-core/src/test/java/co/rsk/peg/lockingcap/LockingCapSupportImplTest.java

```diff
@@ -0,0 +1,302 @@
+
+        Coin newLockingCap = previousLockingCap.add(Coin.SATOSHI);
+        Transaction tx = TransactionUtils.getTransactionFromCaller(signatureCache, LockingCapCaller.AUTHORIZED.getRskAddress());
+
+        // Act
+        boolean actualResult = lockingCapSupport.increaseLockingCap(tx, newLockingCap);
+
+        // Assert
+        assertTrue(actualResult);
+        assertEquals(Optional.of(newLockingCap), lockingCapSupport.getLockingCap());
+    }
+
+    @Test
+    void increaseLockingCap_whenAPreviousValueExistsInStorageAndNewLockingCapIsLessThanPreviousValue_shouldReturnFalse() {
+        // Arrange
+        Coin previousLockingCap = constants.getInitialValue().add(Coin.SATOSHI);
+        lockingCapStorageProvider.setLockingCap(previousLockingCap);
+        lockingCapSupport.save();
+        // Recreate LockingCapSupport to load the previous value from storage
+        lockingCapStorageProvider = new LockingCapStorageProviderImpl(bridgeStorageAccessor);
+        lockingCapSupport = new LockingCapSupportImpl(lockingCapStorageProvider, activations, constants, signatureCache);
+
+        Coin newLockingCap = previousLockingCap.subtract(Coin.SATOSHI);
+        Transaction tx = TransactionUtils.getTransactionFromCaller(signatureCache, LockingCapCaller.AUTHORIZED.getRskAddress());
+
+        // Act
+        boolean actualResult = lockingCapSupport.increaseLockingCap(tx, newLockingCap);
+
+        // Assert
+        assertFalse(actualResult);
+        assertEquals(Optional.of(previousLockingCap), lockingCapSupport.getLockingCap());
+    }
+
+    @Test
+    void increaseLockingCap_whenAnUnauthorizedCallerRequestToIncreaseLockingCapValue_shouldReturnFalse() {
+        // Arrange
+        Coin newLockingCap = constants.getInitialValue().add(Coin.SATOSHI);
+        Transaction tx = TransactionUtils.getTransactionFromCaller(signatureCache, LockingCapCaller.UNAUTHORIZED.getRskAddress());
+
+        // Act
+        boolean actualResult = lockingCapSupport.increaseLockingCap(tx, newLockingCap);
+
+        // Assert
+        assertFalse(actualResult);
+        assertEquals(Optional.of(constants.getInitialValue()), lockingCapSupport.getLockingCap());
+    }
+
+    @Test
+    void increaseLockingCap_whenNewLockingCapIsGreaterThanMaxLockingCap_shouldReturnFalse() {
+        // Arrange
+        Coin previousLockingCap = constants.getInitialValue().add(Coin.SATOSHI);
+        lockingCapStorageProvider.setLockingCap(previousLockingCap);
+        lockingCapSupport.save();
+        // Recreate LockingCapSupport to load the previous value from storage
+        lockingCapStorageProvider = new LockingCapStorageProviderImpl(bridgeStorageAccessor);
+        lockingCapSupport = new LockingCapSupportImpl(lockingCapStorageProvider, activations, constants, signatureCache);
+
+        Coin maxLockingCap = previousLockingCap.multiply(constants.getIncrementsMultiplier());
```

Comment:
This is not the `maxLockingCap`. The max locking cap is 21M, which is the maximum btc amount that will ever exist. 

This is just the limit for the next locking cap increment. I would choose a better name like `nextLimitLockingCap`. Wdyt?

Reply:
The `maximum LockingCap limit` is calculated in this way. If you see the code, we don't have an amount limit to be increased. The `maxLockingCap` is calculated from the current LockingCap and multiplied with `LockingCapConstants.getIncrementsMultiplier()`. Take a look at the code:
https://github.com/rsksmart/rskj/blob/501e7cbaae492e586ad78df0c841de6e8d8dc5cb/rskj-core/src/main/java/co/rsk/peg/lockingcap/LockingCapSupportImpl.java#L71

@marcos-iov might shed some light on this matter.

About the variable name, I'm using the same as the consensus code.

Reply:
I know how it is calculated. My point is regarding the name of the variable. Imo, `maxLockingCap` express `maxRbtc`(21M), which is not the case here. 

I would name it `limitLockingCap` instead.